### PR TITLE
Move MetadataList to elements

### PR DESCRIPTION
**Pull request by Joozty**
I would like to reuse this on on /data-providers and /display pages. In the next PR I will generalise AuthorList component so it can be reused too.

### CODE REVIEW

File: src/elements/metadata-list/metadata-list-item.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { classNames } from 'utils'
+
+const MetaDataListItem = ({
+  children,
+  label,
+  id,
+  className,
+  itemProp = null,
+  itemScope = null,
+  itemType = null,
```
Comment (viktor-yakubiv) [inline src/elements/metadata-list/metadata-list-item.jsx:15]: ~Why don't you leave this implicit? I don't think schema.org should be part of the library even if we use it.~ I see, it's a technical issue.

I am not sure where `htmlProps` should go, to `div` or `dd` 🤔

File: src/elements/metadata-list/metadata-list.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+```jsx
+  <MetadataList>
+    <MetadataList.Item label="Field of study">
+      Computer Science
+    </MetadataList.Item>
+
+    <MetadataList.Item
+      label="Publication data"
+    >
+      23. June 2016
```
Comment (viktor-yakubiv) [inline src/elements/metadata-list/metadata-list.md:10]: Is there a dot needed?
```suggestion
      23 June 2016
```

File: src/elements/metadata-list/metadata-list.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    </MetadataList.Item>
+
+    <MetadataList.Item
+      label="Publisher"
+    >
+      Proceedings of the 16th ACM/IEEE-CS
+       on Joint Conference on Digital Libraries – JCDL '16
```
Comment (viktor-yakubiv) [inline src/elements/metadata-list/metadata-list.md:17]:
```suggestion
      Proceedings of the 16th ACM/IEEE-CS
      on Joint Conference on Digital Libraries – JCDL '16
```

File: src/elements/metadata-list/metadata-list.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+```jsx
+  <MetadataList>
+    <MetadataList.Item label="Field of study">
+      Computer Science
+    </MetadataList.Item>
+
+    <MetadataList.Item
+      label="Publication data"
+    >
```
Comment (viktor-yakubiv) [inline src/elements/metadata-list/metadata-list.md:9]:
```suggestion
    <MetadataList.Item label="Publication data">
```

**Review by viktor-yakubiv — APPROVED** [review]

File: src/elements/metadata-list/metadata-list.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+```jsx
+  <MetadataList>
+    <MetadataList.Item label="Field of study">
+      Computer Science
+    </MetadataList.Item>
+
+    <MetadataList.Item
+      label="Publication data"
```
Comment (Joozty) [inline src/elements/metadata-list/metadata-list.md:8]:
```suggestion
      label="Publication date"
```

File: src/elements/metadata-list/metadata-list.md (hunk 4 above)
Comment (Joozty) [reply src/elements/metadata-list/metadata-list.md]: Maybe we should setup linter for jsx section in md files (if it is possible)

**Pull request merged**